### DISCUSSION TITLE
Cannot get PS4 save to load

### DISCUSSION THREAD

**Discussion by MrTorgue** [zencq/NomNom#88] [Q&A] [answered]
I'm having trouble reading my PS4 save i exported from save wizard.  Did the latest no man's sky update break this feature?  I just bought an XBOX Series X and do not feel like starting over after spending over 1,000 hours on PS4 (and about the same on PC frankly).  The app just shows a strange box instead of the platform (and says Unknown) and every area of the page is blank or greyed out.  

What am I missing? 

![nms](https://user-images.githubusercontent.com/105680167/168690542-273ba773-5c41-46c6-be92-b7bb87457518.png)

**Comment by cengelha** [comment, marked as answer]
Hi! You must select the folder with the savedata*.hg files inside as save path. Then they should be detected. If not, please send the folder (or a screenshot of it first).

The target path in the transfer section is the destination platform (Steam/Microsoft). Select slots for source and destination and make sure LID/UID/USN are set (create new creative save and claim the base if it's not the case).

When all is ready you can click the little icon between "Source" and "Destination".
